crt_portal/cts_forms/views.py

```diff
@@ -20,11 +20,18 @@ class CRTReportWizard(SessionWizardView):
     """once all the sub-forms are submitted this class will clean data and save."""
     template_name = 'forms/report.html'
 
-
     def done(self, form_list, form_dict, **kwargs):
-        form_data = [form.cleaned_data for form in form_list]
-        consolidated_data = dict(ChainMap(*form_data))
-        report_instance = Report(**consolidated_data)
-        report_instance.save()
+        form_data_dict = self.get_all_cleaned_data()
+        m2mfield = form_data_dict.pop('protected_class')
+        r = Report.objects.create(**form_data_dict)
+        r.save()
+
+        for protected in m2mfield:
```

Comment:
@LindsayYoung I think a code comment would be helpful here to explain why custom handling of the form data is needed.

Reply:
Fixed, thanks @LindsayYoung 😄